matRad_generateStf.m

```diff
@@ -237,20 +237,20 @@
                              ct.resolution, ...
                              stf(i).sourcePoint, ...
                              stf(i).ray(j).targetPoint, ...
-                             {voiTarget});
+                             [{ct.cube{1}} {voiTarget}]);
 
         % find appropriate energies for particles
        if strcmp(stf(i).radiationMode,'protons') || strcmp(stf(i).radiationMode,'carbon')
 
            % target hit
-           if sum(rho{1}) > 0 
+           if sum(rho{2}) > 0 
```

Comment:
I remember that there was a reason why we had to introduce this but I am also unsure now if it is really needed. The padding is one option, another option is that we are only using the voxel center coordinates to set up the rays and not an exact volumetric representation of all target voxels.